backend/app/api/endpoints/users.py

```
@@ -633,6 +638,24 @@ async def reorder_git_tokens(
     )
 
 
+@router.get(
+    "/me/git-accounts/sync-summary",
+    response_model=GitAccountSyncSummary,
+)
+async def get_git_account_sync_summary(
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)